src/models/network.js

```diff
@@ -106,7 +106,7 @@ Network.prototype.validate = function (client) {
 	this.host = cleanString(this.host).toLowerCase();
 	this.name = cleanString(this.name);
 	this.saslAccount = cleanString(this.saslAccount);
-	this.saslPassword = cleanString(this.saslPassword);
+	this.saslPassword = this.saslPassword.replace(/[\x00\r\n]/g, ""); // Clean but don't truncate
```

Comment:
Hm... we should probably not open code this here, but rather split the cleanString method into cleanString and cleanStringAndTruncate.
Problem being that should this ever change you now need to fix it twice.

Then again, does this ever change?

Reply:
We can also test this + cleanString if we split it into a new function and move them to a new file/export them, but that may be overkill for a fairly simple regex.

Reply:
What I would do is add length param to `cleanString`, and just provide a higher length for password (I mean right now do people seriously have passwords longer than 300 chars? that's already silly).